cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -728,12 +728,12 @@ func StartController(ctx context.Context, controllerCtx ControllerContext, contr
 
 	time.Sleep(wait.Jitter(controllerCtx.ComponentConfig.Generic.ControllerStartInterval.Duration, ControllerStartJitter))
 
-	logger.V(1).Info("Starting controller", "controller", controllerName)
+	logger.V(1).Info("Initializing controller", "controller", controllerName)
```

Comment:
The though behind this is that we do initialization+starting here: `GetInitFunc`/`initFunc`. And the start itself will be logged by each controller.